src/API/Controllers/EmployeeController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Core.Interfaces;
+using Core.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace API.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmployeeController : ControllerBase
+    {
+        private readonly IEmployeeRepo _repository;
+
+        public EmployeeController(IEmployeeRepo repo)
+        {
+            _repository = repo;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetEmployees()
+        {
+            var employees = await _repository.GetEmployeesAsync();
```

Comment:
[Dotyczy wszystkich kontrolerów] ASP udostępnia CancelationToken `HttpContext.RequestAborted`, dobrą praktyką jest go podawać w dół.
Dzięki temu:
* Jak klient urwie request(kod HTTP 499) utniesz przetwarzanie od  po swojej stronie odzyskując połączenie do Db, wątek w threadpoolu itp z zasobami.
* Jak będziesz używać rate-limittingu to umożliwi to jego działanie



https://learn.microsoft.com/en-us/aspnet/core/fundamentals/use-http-context?view=aspnetcore-9.0#requestaborted